src/main/java/com/commercetools/sync/services/impl/CategoryServiceImpl.java

```diff
@@ -48,7 +51,15 @@ public CategoryServiceImpl(@Nonnull final CategorySyncOptions syncOptions) {
         }
 
         final Consumer<List<Category>> categoryPageConsumer = categoriesPage ->
-            categoriesPage.forEach(category -> keyToIdCache.put(category.getKey(), category.getId()));
+            categoriesPage.forEach(category -> {
```

Comment:
is this class/method specific only for target/source category? If yes, then you can add the name to be e.g. `targetCategory`.

Reply:
The sync always takes as an input a list of drafts. It never manipulates with the source project. Therefore, the entire services of the whole project are used for manipulating the target CTP project. I think renaming it to `targetCategory` would only mean that the service might also be extended to be used for a the source project which it shouldn't.